apps/app/src/views/routes/Settings/components/users/EditRolesAction.tsx

```diff
@@ -94,11 +94,7 @@ const roleSchema = yup
             street1: yup.string().required('Address is required'),
             street2: yup.string(),
             city: yup.string().required('City is required'),
-            state: yupStateSchema.when('roles', {
-              is: (roles: { value: string; label: string }[]) => hasPrescriberRole(roles),
-              then: yup.string().required('Please enter a valid state'),
-              otherwise: yup.string()
-            }),
+            state: yupStateSchema,
```

Comment:
@photonsFardad I can't remember why this was changed:
https://github.com/Photon-Health/client/pull/953

Reply:
I dont exactly remember. I think it had something to do with state is required when updating when its a prescriber role